src/pages/Community/CommunityForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import * as C from './CommunityStyle';
+import { IoMdArrowDropdown } from 'react-icons/io';
+import CommunityHashTag from './CommunityHashTag';
+
+export default function CommunityForm({ findedItem }) {
+  const [categoryBox, setCategoryBox] = useState();
+  const [categoryList, setCategoryList] = useState('CATEGORY');
+  const [hashTagItems, setHashTagItems] = useState([]);
+  const [content, setContent] = useState('');
+  const [hashTag, setHashTag] = useState(['#']);
+  const handleHashTag = e => {
+    e.preventDefault();
+    setHashTag(e.target.value);
+  };
+  const hadlecontent = e => {
+    setContent(e.target.value);
+    findedItem.content = e.target.value;
+  };
+
+  const addHashTag = e => {
+    e.preventDefault();
+    if (hashTagItems.length > 5) {
+      return;
+    }
+    if (e.key === 'Enter' && e.nativeEvent.isComposing === false) {
+      setHashTagItems([hashTag, ...hashTagItems]);
+      setHashTag('#');
+      findedItem.hashTag = [hashTag, ...findedItem.hashTag];
+    }
+  };
+  const selected = () => {
+    setCategoryBox(!categoryBox);
+  };
```

Comment:
```suggestion
  const selected = () => {
    setCategoryBox(prev => !prev);
  };
```
boolean 값이라면 이렇게 함수형 업데이트를 하는 것이 더 안전한 방법입니다!